### PR TITLE
Prep release 3.0.0

### PR DESCRIPTION
**Pull request by jan--f**

### CODE REVIEW

**Review by ArthurSens — APPROVED** [review]
:shipit:

**Review by Nexucis — APPROVED** [review]
🥳 🎉 🎉 🎉 

So cool to see this finally happening !!!

**Review by SuperQ — APPROVED** [review]
🥳

**Review by bboreham — APPROVED** [review]

File: docs/migration.md [hunk 1]
```diff
@@ -207,6 +207,12 @@ This should **only** be applied to metrics that currently produce such labels.
         regex: (\d+)\.0+;.*_bucket
 ```
 
+### Disallow configuring Alertmanager with the v1 API
+Prometheus 3 no longer supports Alertmanagers v1 API. Effectively Prometheus 3
```
Comment (fionaliao) [inline docs/migration.md:211]:
```suggestion
Prometheus 3 no longer supports Alertmanager's v1 API. Effectively Prometheus 3 
```

File: docs/migration.md [hunk 2]
```diff
@@ -207,6 +207,12 @@ This should **only** be applied to metrics that currently produce such labels.
         regex: (\d+)\.0+;.*_bucket
 ```
 
+### Disallow configuring Alertmanager with the v1 API
+Prometheus 3 no longer supports Alertmanagers v1 API. Effectively Prometheus 3 
+requires [Alertmanager 0.16.0](https://github.com/prometheus/alertmanager/releases/tag/v0.16.0) or later. User with older Alertmanager
```
Comment (fionaliao) [inline docs/migration.md:212]:
```suggestion
requires [Alertmanager 0.16.0](https://github.com/prometheus/alertmanager/releases/tag/v0.16.0) or later. Users with older Alertmanager
```

**Comment by tjhop** [comment]
:fire: :fire: :fire:

**Comment by ywwg** [comment]
🚀 🚀 🚀 🚀 🚀

**Review by fionaliao — APPROVED** [review]

**Review by bwplotka — APPROVED** [review]
♥️

**Review by gotjosh — APPROVED** [review]
LGTM

**Review by carrieedwards — APPROVED** [review]
:rocket:

**Pull request merged**